repos/system_upgrade/el7toel8/actors/localreposinhibit/actor.py

```diff
@@ -0,0 +1,72 @@
+from leapp import reporting
+from leapp.actors import Actor
+from leapp.models import TMPTargetRepositoriesFacts, UsedTargetRepositories
+from leapp.reporting import Report
+from leapp.tags import IPUWorkflowTag, TargetTransactionChecksPhaseTag
+from leapp.utils.deprecation import suppress_deprecation
+
+
+@suppress_deprecation(TMPTargetRepositoriesFacts)
+class LocalReposInhibit(Actor):
+    """Inhibits the upgrade if local repositories were found."""
+
+    name = "local_repos_inhibit"
+    consumes = (
+        UsedTargetRepositories,
+        TMPTargetRepositoriesFacts,
+    )
+    produces = (Report,)
+    tags = (IPUWorkflowTag, TargetTransactionChecksPhaseTag)
+
+    def file_baseurl_in_use(self):
+        """Check if any of target repos is local.
```

Comment:
Just reminder, that the style used in the majority of code in leapp repositories is:
```
"""
Check if any....
"""

Reply:
It's not such a big deal, but you're right. It's covered in our python coding guidelines: https://leapp.readthedocs.io/en/latest/python-coding-guidelines.html#docstrings.

Reply:
It's actually inconsistent in this one string

    """Check if any
    
    Lorem ipsum blah blah..
    """

Reply:
This is the most basic python convention for multiline docstrings. Why do you need to irritate people with this exception? A lot of IDEs utilities rely on this convention. As well as the sphinx autodoc. I acknowledge that this is now in our guidelines, so my current message is a kind of product of emotions.

Reply:
If I recall correctly the reasoning back then was related to the fact that the first line would have no whitespaces before the sentence but all the other lines of the multiline string would have X number of whitespaces. We were planning to parse it or already do parse it (maybe for the [dashboard](https://oamg.github.io/leapp-dashboard/), @vinzenz?) and this whitespace inconsistency could cause troubles.

Reply:
I guess that the `strip` of the string could resolve such problems if we have any, but not sure as I am not familiar with python parser. My point is mainly about the `aesthetics`. And the majority of our code uses the new line in docstring, so to be consistent. I do not see so big benefit to change our coding guidelines to be honest. One more enter per docstring is not something what seems too much annoying in editor from my POV. And even when I am fun of the newline start in docstring (it really looks much better..) I am not telling that this is blocker. Just it looks really weird from my POV.

Reply:
Btw, one of topics about that I remeber was that many people want to put empty space before and after the docstring
```
""" Docstring. """
```
Using the new line always automatically resolves this problem and you have not to write people they have to remove the whitespaces around the docstring.

Reply:
If it is not breaking something in dashboard (@vinzenz ), then I suggest closing this thread in a separate ticket, because if I am testing now standard flake8 pre-commit hooks, with the docstrings in the following form
```
"""
Some sdsdsd
sdsdsds
""""
```

I see the error:
```
D205 1 blank line required between summary line and description
D400 First line should end with a period
```
So the user won't be able to commit the wrong form of the docstrings

Reply:
Yes, but it's because it should be correctly like this:
```
Some sdsdsd.

sdsdsds
```
right?

Reply:
@pirat89 now I see, you're right. So the docstrings linter accepts your variant and I'm really surprised about it. So both versions work ok:
```
"""Summary.

details
"""
```
and
```
"""
Summary.

details
"""
```